pages/common/drupal.md

```diff
@@ -0,0 +1,21 @@
+# drupal
+
+> Drupal Console.
+> The Drupal CLI. A tool to generate boilerplate code, interact with and debug Drupal.
+> More information: <https://drupalconsole.com/>.
+
+- Install module "foo":
```

Comment:
Perhaps it's better to say "Install module:" and in the example, change it to "module_name" from "foo". And do the same for next one too.

Reply:
I tried to use the same syntax as the `drush` page. But I agree `{{module_name}}` is easier to understand.